src/main/java/oop/beverages/Latte.java

```diff
@@ -0,0 +1,27 @@
+package oop.beverages;
+
+public final class Latte extends Coffee {
+    public Latte(String Flavor) {
+        this.setFlavor(Flavor);
+    }
+
+    private void setFlavor(String flavor) {
```

Comment:
this implementation is not correct it should be ...

public void setFlavor(String flavor) {
    this.flavor = flavor
}

but the real issue is that this method doesn't belong here -- if flavor is defined in Coffee.java, you only need the setter method there!